ensime_shared/client.py

```diff
@@ -224,26 +202,10 @@ def tell_module_missing(self, name):
         msg = feedback["module_missing"]
         self.editor.raw_message(msg.format(name, name))
 
-    # TODO: this should be the Ensime class's responsibility, not EnsimeClient;
-    #       move (and eliminate self.vim) after fixing, see #294
-    def disable_plugin(self):
-        """Disable plugin temporarily, including also related plugins."""
-        self.log.debug('disable_plugin: in')
-
-        def threadsafe_vim(command):
-            """Threadsafe call if neovim, normal if vim."""
-            def normal_vim(e):
-                self.vim.command(command)
-            with catch(Exception, normal_vim):
-                self.vim.session.threadsafe_call(command)
```

Comment:
Oh that's interesting, I'd seen this error in testing but never ran into it in day-to-day usage (I still don't use Neovim 100% of the time). This change seemed correct but I wasn't quite sure because it didn't work as I expected with [REPL testing][1]. Good to know it indeed appears to be working.

[1]: https://github.com/neovim/python-client#usage-through-the-python-repl